foqus_lib/framework/graph/node.py

```diff
@@ -106,15 +109,33 @@ def __init__(self, model):
         for i in range(np.shape(self.model.inputs[0])[1]):
             try:
                 input_label = self.model.layers[1].input_labels[i]
-            except:
+            except AttributeError:
+                logging.getLogger("foqus." + __name__).info(
```

Comment:
I think these extended messages are great, they add lots of useful context both for users and other developers.

One minor suggestion: instead of repeating the `logging.getLogger("foqus." + __name__)` each time, you could create a logger object at the module level, e.g. `_logger = logging.getLogger("foqus." + __name__)` after the `import` statements on Line 40, and then use that object throughout the rest of the module, i.e. `_logger.info(...)`